record_keeping_wizard/models/matter_wizard.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class MatterWizard(models.TransientModel):
+    _name="rk.wizard"
+
+    def _get_model(self):
+        raise NotImplementedError("Expects this function to be implemented in inheritance.")
+
+    # TODO: In inheritence, implement field 'model'
```

Comment:
TODO -> Remember
Implement 'model' as property and throw errors when not overwriten if used